packages/components/src/Table/Body/TableBody.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import TableRow, { getRowId } from '../Row/TableRow.js';
+import theme from './Body.scss';
+
+function getBodyClassName(classNameProvider) {
+	if (classNameProvider && classNameProvider.getForBody) {
+		return classNameProvider.getForBody();
+	}
+	return '';
+}
+
+/**
+ * This function is responsible for rendering an element in the table.
+ */
+function renderRow(
```

Comment:
You don't need this function, just put the resulting jsx in your elements.map()

Reply:
ok